src/pages-and-resources/live/data/api.js

```diff
@@ -0,0 +1,165 @@
+  const apps = Object.entries(data.available).map(([key, app]) => ({
+    id: key,
+    messages: app.messages,
+    externalLinks: {
+      learnMore: app.external_links.learn_more,
+      configuration: app.external_links.configuration,
+      general: app.external_links.general,
+      accessibility: app.external_links.accessibility,
+      contactEmail: app.external_links.contact_email,
+    },
+    adminOnlyConfig: !!app.admin_only_config,
+  }));
+  return {
+    activeAppId: data.active,
+    apps,
+  };
+}
+
+function normalizeSettings(data) {
+  return {
+    appConfig: normalizeAppConfig(data),
+    piiConfig: normalizePiiSharing(data.lti_configuration),
+  };
+}
+
+function denormalizeData(courseId, appId, data) {
+  const ltiConfiguration = {};
+
+  if (data.consumerKey) {
+    ltiConfiguration.lti_1p1_client_key = data.consumerKey;
+  }
+  if (data.consumerSecret) {
+    ltiConfiguration.lti_1p1_client_secret = data.consumerSecret;
+  }
+  if (data.launchUrl) {
+    ltiConfiguration.lti_1p1_launch_url = data.launchUrl;
+  }
+  if (data.launchEmail) {
+    ltiConfiguration.lti_1p1_launch_email = data.launchEmail;
+  }
+  if ('piiShareUsername' in data) {
+    ltiConfiguration.pii_share_username = data.piiShareUsername;
+  }
+  if ('piiShareEmail' in data) {
+    ltiConfiguration.pii_share_email = data.piiShareEmail;
+  }
+
+  if (Object.keys(ltiConfiguration).length > 0) {
+    // Only add this in if we're sending LTI fields.
+    // TODO: Eventually support LTI v1.3 here.
+    ltiConfiguration.version = 'lti_1p1';
+  }
+
+  const apiData = {
+    context_key: courseId,
+    enabled: true,
+    lti_configuration: ltiConfiguration,
+    provider_type: appId,
+  };
+
+  return apiData;
+}
+
+export function getLiveProvidersUrl() {
+  // return `${getConfig().STUDIO_BASE_URL}/api/live/v0/course/${courseId}/providers`;
+}
+
+export function getLiveSettingsUrl() {
+  // return `${getConfig().STUDIO_BASE_URL}/api/live/v0/course/${courseId}/settings`;
+}
+
+export async function getLiveProviders() {
+  // const { data } = await getAuthenticatedHttpClient()
+  //   .get(getLiveProvidersUrl(courseId));
+
+  const data = {
+    active: 'zoom',
+    available: {
+      zoom: {
+        admin_only_config: true,
+        external_links: {
+          accessibility: '',
+          configuration: '',
+          contact_email: '',
+          general: 'https://edstem.org/us/',
+          learn_more: '',
+        },
+        messages: [
+          'Schedule meetings with Zoom to conduct live course sessions with learners',
+        ],
+        supports_lti: true,
+      },
+    },
+  };
+  return normalizeProviders(data);
+}
+
+export async function getLiveSettings(courseId, providerId = null) {
+  const params = {};
+  if (providerId) {
+    params.params = { provider_id: providerId };
+  }
+  // const url = getLiveSettingsUrl(courseId);
+  // const { data } = await getAuthenticatedHttpClient()
+  //   .get(url, params);
```

Comment:
if that will be left on purpose for API integration if not then please remove commented code.

Reply:
It is left of puporse for API integration.